executor/http_runner.go

```diff
@@ -100,6 +100,7 @@ func (f *HTTPFunctionRunner) Run(req FunctionRequest, contentLength int64, r *ht
 	}
 
 	request.Host = r.Host
+	request.ContentLength = r.ContentLength
```

Comment:
Not sure we should set this. If this is content-type chunked it will have unintended results.